app/src/main/java/com/github/wnder/user/UserDatabaseUtils.java

```diff
@@ -0,0 +1,89 @@
+package com.github.wnder.user;
+
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Utils for the user database class
+ */
+public class UserDatabaseUtils {
+    private CompletableFuture<List<String>> guessedPics;
+    private CompletableFuture<Set<Double>> scores;
+
+    /**
+     * Constructor
+     * @param guessedPics future of all user guessed pics
+     * @param scores future of all user scores
+     */
+    public UserDatabaseUtils(CompletableFuture<List<String>> guessedPics, CompletableFuture<Set<Double>> scores){
+        this.guessedPics = guessedPics;
+        this.scores = scores;
+    }
```

Comment:
We have talked about it orally, it would mean quite big changes on the overall architecture. The problem will be hard to solve right know, so it's better to let it like this.